apps/docs/spec/cli_v1_config.yaml

```diff
@@ -1254,6 +1287,28 @@ parameters:
       - name: 'Auth Multi-Factor Authentication'
         link: 'https://supabase.com/docs/guides/auth/auth-mfa'
 
+  - id: 'auth.sessions.timebox'
+    title: 'auth.sessions.timebox'
+    tags: ['auth']
+    required: false
+    default: '24h'
+    description: |
+      Force log out after the specified duration. Sample values include: '50m', '20h'.
+    links:
+      - name: 'Auth Sessions'
+        link: 'https://supabase.com/docs/guides/auth/sessions'
+
+  - id: 'auth.sessions.inactivity_timeout'
+    title: 'auth.sessions.inactivity_timeout'
+    tags: ['auth']
+    required: false
+    default: '8h'
```

Comment:
Is it really default 8h?

Reply:
Good catch, it's not, 8h is the example from testdata. Updating now..